neard/src/config.rs

```diff
@@ -140,21 +139,19 @@ pub const VALIDATOR_KEY_FILE: &str = "validator_key.json";
 pub const MAINNET_TELEMETRY_URL: &str = "https://explorer.mainnet.near.org/api/nodes";
 pub const NETWORK_TELEMETRY_URL: &str = "https://explorer.{}.near.org/api/nodes";
 
-lazy_static! {
-    /// The rate at which the gas price can be adjusted (alpha in the formula).
-    /// The formula is
-    /// gas_price_t = gas_price_{t-1} * (1 + (gas_used/gas_limit - 1/2) * alpha))
-    pub static ref GAS_PRICE_ADJUSTMENT_RATE: Rational = Rational::new(1, 100);
+/// The rate at which the gas price can be adjusted (alpha in the formula).
+/// The formula is
+/// gas_price_t = gas_price_{t-1} * (1 + (gas_used/gas_limit - 1/2) * alpha))
+pub const GAS_PRICE_ADJUSTMENT_RATE: Rational = Rational::new_raw(1, 100);
 
-    /// Protocol treasury reward
-    pub static ref PROTOCOL_REWARD_RATE: Rational = Rational::new(1, 10);
+/// Protocol treasury reward
+pub const PROTOCOL_REWARD_RATE: Rational = Rational::new_raw(1, 10);
 
-    /// Maximum inflation rate per year
-    pub static ref MAX_INFLATION_RATE: Rational = Rational::new(5, 100);
+/// Maximum inflation rate per year
+pub const MAX_INFLATION_RATE: Rational = Rational::new_raw(5, 100);
 
-    /// Protocol upgrade stake threshold.
-    pub static ref PROTOCOL_UPGRADE_STAKE_THRESHOLD: Rational = Rational::new(8, 10);
-}
+/// Protocol upgrade stake threshold.
+pub const PROTOCOL_UPGRADE_STAKE_THRESHOLD: Rational = Rational::new_raw(8, 10);
```

Comment:
ah so this should be changed to `Rational::new_raw(4, 5)`, because `Rational::new` does normalization it to 4,5